debezium-core/src/main/java/io/debezium/transforms/UnwrapFromEnvelope.java

```diff
@@ -115,9 +119,19 @@ public static DeleteHandling parse(String value, String defaultValue) {
                     + "drop - records are removed,"
                     + "rewrite - __deleted field is added to records.");
 
+    private static final Field OPERATION_HEADER = Field.create("operation.header")
+            .withDisplayName("Adds the debezium operation into the message header")
```

Comment:
Minor nit: I don't think it's _into the_ header, but rather _as a_ header. I.e. there can be multiple headers, and we add one.